shared/src/lib/utils.js

```diff
@@ -0,0 +1,92 @@
+export async function summarizeContent({
+  url,
+  text,
```

Comment:
I added support for [the `text` param](https://help.kagi.com/kagi/api/summarizer.html#parameters) here, but nothing in the API is calling it yet. That will come later, as I'll need to also build a separate UI to allow the extension to close but still finish the request and show the response of the Summarizer.